app/views/kernel_configs/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with(model: kernel_config) do |form| %>
```

Comment:
This view, by itself, is not useful because we want to create configs as part of a build.  I'm fine with this for now but once we have the build interface more fleshed out it will likely be that this view is deleted.

Reply:
Understood and I thought of that too. My plan was to have a shared `_form` partial that connected all the pieces. Or, alter the scaffold `KernelBuild` view to account for that.